### ISSUE TITLE
BUG : Fatal Exception: java.util.ConcurrentModificationException

### ISSUE THREAD

**Issue by iamwent** [rudderlabs/rudder-sdk-android#327]
SDK version: v1.18.1

```
Fatal Exception: java.util.ConcurrentModificationException:
       at java.util.HashMap$HashIterator.nextNode(HashMap.java:1574)
       at java.util.HashMap$ValueIterator.next(HashMap.java:1602)
       at kotlin.collections.CollectionsKt___CollectionsKt.toList(_Collections.kt:1315)
       at com.rudderstack.android.repository.RudderDatabase$init$1$2._init_$lambda$1(RudderDatabase.kt:84)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
       at java.lang.Thread.run(Thread.java:1012)
```

**Comment by itsdebs** [comment]
@iamwent this is fixed in 1.20.0

**Comment by itsdebs** [comment]
I hope the issue is fixed. Please reopen in case the issue resurfaces.
